inventory/any.cf

```diff
@@ -154,6 +159,58 @@ bundle agent cfe_autorun_inventory_memory
 
 }
 
+bundle agent cfe_autorun_inventory_setuid
+# @brief Inventory setuid files and prune invalid entries from the setuid log
+{
+  vars:
+    !disable_inventory_setuid::
+      "candidates" slist => lsdir( "$(sys.workdir)", "cfagent\..*\.log", true );
+
+      "setuid_log_path"
+        comment => "We select the file that matches the downcased version of the
+                    hostname since sys.fqhost always returns lower case",
+        string => "$(candidates)",
+        if => strcmp( "$(sys.workdir)/cfagent.$(sys.fqhost).log",
+                      string_downcase($(candidates)));
+
+      "files" slist => readstringlist( $(setuid_log_path), "", "$(const.n)", inf, inf);
+
+      "setuid[$(files)]"
+        string => "$(files)",
+        meta => { "inventory", "attribute_name=setuid Files" },
```

Comment:
@craigcomstock consistency hobgoblin ....

I guess this should be `Setuid files`